spec/gameobj-data/boxes_and_picking_spec.rb

```diff
@@ -161,6 +161,17 @@
       end
     end
 
+    describe "legendary loot boxes" do
+      [
+        %{filigreed rolaren reliquary},
+      ].each do |box_desc|
+        it "recognizes #{box_desc} as a box" do
+        box = GameObjFactory.item_from_name(box_desc)
+          expect(box.type).to include "box"
```

Comment:
This will be "uncommon" as well because it includes rolaren in the name. I think that's fine but you can add an exclude if you think it makes sense to remove it.